support/src/main/java/se/inera/intyg/common/support/facade/util/PatientToolkit.java

```diff
@@ -23,15 +23,24 @@
 
 public final class PatientToolkit {
 
+    private static final int[] SUBSTRING_INDEX = {0, 4, 6, 8};
```

Comment:
A thought... not sure that this constant array improves clarity to the code. If you like to have constants (which makes sense) instead of hardcoding the indexes, maybe think about splitting the array into four separate constants and give them meaningful names?